modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -695,6 +694,37 @@ include::mule-runtime:ROOT:page$mmp-concept.adoc[tag=deploymentTimeoutParameterD
 include::mule-runtime:ROOT:page$mmp-concept.adoc[tag=skipParameterDescription]
 |===
 
+== Deploy a Domain
+
+The Mule Maven Plugin supports deploying domains to standalone Mule instances and also when using the Mule agent.
+
+To deploy a domain, use the same configuration and deployment steps you use when deploying an application. For example, to deploy a domain to a Standalone instance:
```

Comment:
to deploy a domain to a Standalone instance > to deploy a domain to a standalone instance: